components/calendar/src/japanese.rs

```diff
@@ -158,6 +158,7 @@ const REIWA_START: EraStartDate = EraStartDate {
     day: 1,
 };
 
+#[allow(clippy::panic)] // TODO(#NNN) Clippy exceptions need docs or fixing.
```

Comment:
suggestion: I think these errors are coming from the `tinystr!` macro implementation. I would suggest adding this suppression in the `utils/tiny_str` crate.

Reply:
+1. It looks like a large fraction of these come from the `tinystr!` macro.